.github/workflows/publish-dist.yml

```diff
@@ -31,9 +31,10 @@ jobs:
           set -ex
           mkdir dist-repo
           cd dist-repo
-          tar -xf ../artifact/webpack-jumpstart.tar dist package-lock.json
+          tar -xf ../artifact/webpack-jumpstart.tar dist package-lock.json .sha256sums || true
```

Comment:
Do you already have the follow-up for this in the pipeline? I don't want that alternative truth in production for any extended time (> 1 day).

Reply:
#16005

Reply:
For the record, I think this "alternative truth" is safe.